docs/source/contents/4_draft_mapping.rst

```diff
@@ -1777,10 +1777,644 @@ Decision
+
+`helser:24792 as an example of mods:subject/mods:name[@valueURI] <https://digital.lib.utk.edu/collections/islandora/object/hesler%3A24792/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n87116131">
+            <namePart>Atkinson, George Francis, 1854-1918</namePart>
+        </name>
+    </subject>
+    <subject>
+        <name authority="naf" valueURI="http://id.loc.gov/authorities/names/n88144876">
+            <namePart>Arthur, Joseph Charles, 1850-1942</namePart>
+        </name>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/names/n88144876" ;
+        dcterms:subject "http://id.loc.gov/authorities/names/n87116131" .
+
+Name and topical subjects without URIs
+--------------------------------------
+
+Use Case
+^^^^^^^^
+
+We'll need to treat any of these subjects that aren't able to be reconciled as string values. For the postcard collection,
+the use of dots (Database of the Smokies) as the authority makes it impossible to include a URI presently. Other collections
+with string values that are: Charlie Daniel Cartoon Collection, Ed Gamble Cartoon Collection, Football Programs, Insurance Company of
+North America Records, the American Civil War Collection, Ramsey Family Papers, Tennessee Documentary History,
+and Volunteer Voices.
+
+The Volunteer Voices collection includes subjects with three different displayLabel values - Volunteer Voices Curriculum Topics,
+Tennessee Social Studies K-12 Eras in American History, and Broad Topics. These subjects are currently given separate
+facets in Islandora's metadata display. Discovery to the collection via two of these subject categories is also featured
+on the `Tennessee State Library and Archives website <https://sos.tn.gov/products/tsla/volunteer-voices>`_ (Broad Topics
+and Tennessee Social Studies K-12 Eras in American History). While these subjects have been distinguished previously from
+other subjects in the past by their distinct Xpath, having so many different types of subjects was found to be unnecessary
+going forward. Broad Topics and Curriculum Topics will be folded in with all other subjects. For links to external websites,
+like TSLA's, we can use the string values to supply a link without needing to place them in a separate property. Note that
+subjects associated with Tennessee Social Studies K-12 Eras in American History are dealt with
+separately below.
+
+Justification
+^^^^^^^^^^^^^
+
+Subject values are important access points for users that require migration. While URIs would be ideal from a technical
+standpoint, strings still support discovery.
+
+Xpath
+^^^^^
+
+    :code:`mods/subject[not(@valueURI)]/topic[not(@valueURI)]` OR
+    :code:`mods/subject[not(@valueURI)]/name[not(valueURI)]/namePart[not(valueURI)]` OR
```

Comment:
There is an extra OR, but I don't care.

Reply:
Don't care, but in case you do, recommenting.

Reply:
I removed it - thank you for catching it.